pegasus/helpers/section_api_helpers.rb

```diff
@@ -297,7 +297,7 @@ def self.create(params)
       )
     rescue Sequel::UniqueConstraintViolation
       tries += 1
-      retry if tries < 2
+      retry if tries < 3
```

Comment:
Of course, if so, then this retry logic is no longer necessary here. In particular, I can't imagine us wanting to call `SectionHelpers` for a random code that may already exist.